packages/memory/src/utils/buffer-to-uint8array.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Convert a NodeJS Buffer to a JS Uint8Array
+ *
+ * @param buffer NodeJS Buffer
+ * @returns JS Uint8Array
+ */
+export const bufferToUint8Array = (buffer: Buffer) =>
+	new Uint8Array(buffer.buffer, buffer.byteOffset, buffer.byteLength);
```

Comment:
Not very familiar with buffer conversion but wouldn't a simple `new Uint8Array(buffer)` be equal to that?

Reply:
You'd think so but no!

Reply:
![js](https://github.com/directus/directus/assets/5363448/e6c0aa3c-8ee4-408c-bc49-41548b60291e)